collections_config.toml.jinja

```diff
@@ -33,16 +33,6 @@ root_path = "{{ KBCOLL_SERVICE_ROOT_PATH or "" }}"
 # their sharding preferences.
 create_db_on_startup = "{{ KBCOLL_CREATE_DB_ON_STARTUP or "false" }}"
 
-# Set to "true" to not connect to any external services except for authorization,
-# including the database. This will cause all calls to the service that require external
-# service access to fail but is useful for quickly checking OpenAPI documentation
-# or general methods that don't access external services.
-# The service can be run via:
-# docker run --env-file <your .env file> --publish 5000:5000 <image hash>
-# Example:
-# docker run --env-file collections.env --publish 5000:5000 3b6523ebf6b8
-dont_connect_to_external_services = "{{ KBCOLL_DONT_CONNECT_TO_EXT_SERVICES or "false" }}"
```

Comment:
this was useful at the beginning of development but now I never use and docker-compose works better anyway.